GenderPayGap.WebUI/Models/AdminReferenceData/ReferenceDataHelper.cs

```diff
@@ -72,7 +73,7 @@ internal static class ReferenceDataHelper {
             }
             catch (Exception ex)
             {
-                errorMessage = "The selected file could not be uploaded � try again.";
+                errorMessage = "The selected file could not be uploaded � try again.";

```

Comment:
This registers for me as a change, something to do with your character encoding?

Reply:
Weird - it doesn't show as a change for me in GitKraken...

Reply:
Should be fixed now!